src/alitra/frame_dataclasses.py

```diff
@@ -9,32 +9,37 @@
 
 @dataclass
 class Transform:
-    """A transform object that describe the transformation between two frames.
-    Translations must be expressed in the (to_) frame"""
+    """
+    A transform object that describe the transformation between two frames.
+    Euler or quaternion must be provided to perform a rotation. If no rotation is required specify the unit quaternion
+    or zero Euler angles. Translations must be expressed in the (to_) frame
+    """
 
     translation: Translation
-    euler: Euler
     from_: Literal["robot", "asset"]
     to_: Literal["asset", "robot"]
-    rotation_object: Rotation = Rotation.from_euler("zyx", [0, 0, 0])
+    rotation_object: Rotation = None
+    euler: Euler = None
+    quaternion: Quaternion = None
```

Comment:
Elected a solution where both are `None` by default and you must specify exactly one of them as the basis for the rotation.

Reply:
I think this works for our purposes. 

However,  it might be a source of confusion. If we are removing the "from/to" check for angles, then I wonder if we should 
remove both of these, and just initialize with the rotation object itself?.